tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -134,3 +128,220 @@ def test_positive_deploy_configure_by_script(session, form_data):
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        deploy_configure_by_command(config_command)
+        assert get_configure_option('VIRTWHO_DEBUG', VIRTWHO_SYSCONFIG) == '1'
+        session.virtwho_configure.edit(name, {'debug': False})
+        deploy_configure_by_command(config_command)
+        assert get_configure_option('VIRTWHO_DEBUG', VIRTWHO_SYSCONFIG) == '0'
+        session.virtwho_configure.delete(name)
+        assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_interval_option(session, form_data):
+    """ Verify interval dropdown options and the value changes of VIRTWHO_INTERVAL.
+
+    :id: 412a01eb-ae9c-4b23-aeb5-2595873b27e2
+
+    :expectedresults:
+        VIRTWHO_INTERVAL can be changed in /etc/sysconfig/virt-who if the
+        dropdown option is selected to Every 2/4/8/12/24 hours, Every 2/3 days.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        intervals = {
+            'Every hour': '3600',
+            'Every 2 hours': '7200',
+            'Every 4 hours': '14400',
+            'Every 8 hours': '28800',
+            'Every 12 hours': '43200',
+            'Every 24 hours': '86400',
+            'Every 2 days':  '172800',
+            'Every 3 days':  '259200',
+        }
+        for option, value in sorted(intervals.items(), key=lambda item: int(item[1])):
+            session.virtwho_configure.edit(name, {'interval': option})
+            deploy_configure_by_command(config_command)
+            assert get_configure_option('VIRTWHO_INTERVAL', VIRTWHO_SYSCONFIG) == value
+        session.virtwho_configure.delete(name)
+        assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_hypervisor_id_option(session, form_data):
+    """ Verify Hypervisor ID dropdown options.
+
+    :id: dd490f5f-8371-4be6-81d5-a02bbff43de8
+
+    :expectedresults:
+        hypervisor_id can be changed in virt-who-config-{}.conf if the
+        dropdown option is selected to uuid/hwuuid/hostname.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        config_file = get_configure_file(config_id)
+        values = ['uuid', 'hwuuid', 'hostname']
```

Comment:
Again there is no harm to check for each `Hypervisor ID` .. however we are just repeating the same thing in functionality wise .
Can we just keep only one/two from UI (as that also help to check around the same)... and cover everything in way from API/CLI ... your thought on it ?

Reply:
as the above.